src/SafeAreaView.web.tsx

```diff
@@ -19,67 +23,66 @@ const edgeBitmaskMap: Record<Edge, number> = {
   left: LEFT,
 };
 
-export function SafeAreaView({
-  style = {},
-  mode,
-  edges,
-  ...rest
-}: NativeSafeAreaViewProps) {
-  const insets = useSafeAreaInsets();
+export const SafeAreaView: React.ForwardRefExoticComponent<
```

Comment:
Do we need to annotate this type? It should be inferred from forwardRef, and might make the line breaking less aggressive

Reply:
Yes it removes a level of indentation! I thought we wanted to have the type explicitly written like in `SafeAreaView.tsx`.

Reply:
Can you also remove it from SafeAreaView.tsx for consistency?

Reply:
Yes! Done 😉